docker/docker-compose.yaml.jinja2

```diff
@@ -55,18 +55,79 @@ services:
       http_upgrade: '$http_upgrade'
     depends_on:
 {%- for hostname in docker_assembl_hosts %}
-      - assembl{{ loop.index }}{% endfor %}
-    command: /bin/sh -c "cp /opt/default_config/nginx_*.conf > /etc/nginx/conf.d && nginx -g 'daemon off;'"
+      - assembl{{ loop.index }}
+{%- endfor %}
+{%- if docker_sentry == 'docker' %}
+      - sentry
+{%- endif %}
+    command: /bin/sh -c "cp /opt/default_config/nginx_*.conf /etc/nginx/conf.d && nginx -g 'daemon off;'"
     networks:
-      - my_net
+      my_net:
+        aliases:
+{%- for hostname in docker_assembl_hosts %}
+          - {{ hostname }}
+{%- endfor %}
 
 {% if docker_sentry == 'docker' %}
-  # TODO: Configure sentry
   sentry:
     image: sentry:latest
     depends_on:
-      - database
-      - redis
+     - redis
+     - database
+    links:
+     - redis
+     - database:postgres
+    networks:
+      - my_net
+    ports:
+     - 9000:9000
+    stdin_open: true
+    tty: true
+    environment:
+      SENTRY_POSTGRES_HOST: postgres
+      SENTRY_DB_USER: postgres
+      SENTRY_DB_PASSWORD: postgres
+      SENTRY_REDIS_HOST: redis
+      SENTRY_SINGLE_ORGANIZATION: "true"
+      SENTRY_SECRET_KEY: jran5_Hnoherknoudr65_Ryaboeeenh40_Bloboo
+
+  sentry_celery_beat:
+    image: sentry:latest
+    depends_on:
+     - redis
+     - database
+    links:
+     - redis
+     - database:postgres
+    networks:
+      - my_net
+    command: "sentry run cron"
+    environment:
+      SENTRY_POSTGRES_HOST: postgres
+      SENTRY_DB_USER: postgres
+      SENTRY_DB_PASSWORD: postgres
+      SENTRY_REDIS_HOST: redis
+      SENTRY_SINGLE_ORGANIZATION: "true"
+      SENTRY_SECRET_KEY: jran5_Hnoherknoudr65_Ryaboeeenh40_Bloboo
+
+  sentry_celery_worker:
```

Comment:
taken from other example sentry docker-compose.yml files, such as https://gist.github.com/ebuildy/270f4ef3abd41e1490c1
While answering, found this which is more official:
https://github.com/getsentry/onpremise/blob/master/docker-compose.yml